app/assets/stylesheets/shared/dropdown_selector.scss

```diff
@@ -224,7 +227,7 @@
 
     .dropdown-container {
       border-top: 0;
-      box-shadow: 0 1px 2px 0 rgba(0, 0, 0, .2);
+      box-shadow: 0 1px 2px 0 $md-color-shadow-dark;
```

Comment:
Can you change the shadow to look similar to other dropdowns in scinote:
`box-shadow: 0 6px 12px 0 $md-color-shadow-dark;`